Assets/Scripts/Inventory/IItemWithSupply.cs

```diff
@@ -0,0 +1,9 @@
+﻿namespace Inventory
+{
+    public interface IItemWithSupply
+    {
+        void DecreaseSupply();
```

Comment:
Why does `DecreaseSupply` not have an amount when the other one does? And why not just have a single `ChangeSupply`

Reply:
The ItemSupply scriptable object contains the rate of decrease. It is decreased by that amount each time this method is called. I wanted it to be definable on the SO and not handled by each script individually. So you could have special paint that lasts 2x as long or w/e with no code changes required.

As for increase, you can't really get around that, as you can reload 1 bullet or a whole magazine, etc :)